.github/workflows/test.yml

```
@@ -0,0 +1,30 @@
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os:
+          - windows-latest
+          - macos-latest
+          - ubuntu-latest
+        target:
+          - 0.20.2
+          - 1.0.10
+          - 1.2.8
+          - 1.4.0
```

IMO we only need these:

```suggestion
          - 1.0.10
          - 1.4.0
```